packages/nextjs/package.json

```diff
@@ -11,7 +11,8 @@
     "format": "prettier --write . '!(node_module|.next|contracts)/**/*'",
     "check-types": "tsc --noEmit --incremental",
     "vercel": "vercel",
-    "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true"
+    "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true",
+    "spheron": "next build && next export && spheron login --github && spheron publish"
```

Comment:
With the newly added local blockexplorer, `next export` fails since it doesn't work with `getServerSideProps`. So we'll have to think about a potential solution for this.

Maybe in favor of ipfs deployment options (and potentially other deployment options that require `next export`) we could opt to omit the code tab from the blockexplorer while thinkink of a different way to include it (since I think the value of that tab might be rather limited). What do you think @carletex @technophile-04